packages/shared/src/lib/core/network/utils/canAccountMakeStardustTransaction.ts

```diff
@@ -1,18 +1,11 @@
 import { get } from 'svelte/store'
-import { isStardustNetwork } from '@core/network/utils'
 import { activeAccounts } from '@core/profile/stores'
 import { SendFlowType } from '@core/wallet/enums'
-import { NetworkId } from '../types'
 
 export function canAccountMakeStardustTransaction(
     accountIndex: number,
-    networkId: NetworkId,
     sendFlowType: SendFlowType
 ): boolean | undefined {
```

Comment:
This isn't updated in `SelectTokenView`. The check inhibits from runtime errors, so it might be worth keeping.